File: Python/marketplace/create_transfer.py
from pagarmecoreapi.pagarmecoreapi_client import PagarmecoreapiClient
from pagarmecoreapi.models import *
from pagarmecoreapi.controllers import *
from pagarmecoreapi.exceptions.error_exception import *

# Configuration parameters and credentials
basic_auth_user_name = 'basic_auth_user_name' # The username to use with basic authentication
basic_auth_password = 'basic_auth_password' # The password to use with basic authentication

PagarmecoreapiClient(basic_auth_user_name, basic_auth_password)

recipients_controller = recipients_controller.RecipientsController()

request = create_transfer_request.CreateTransferRequest()
request.amount = 100

recipientId = "rp_RElaP4NMCJu08V9m"

try:
    result = recipients_controller.create_transfer(recipientId, request)
    assert result is not None
    print("Transfer id: ", result.id)
    print("Transfer status: ", result.status)
except ErrorException as ex:
    print(ex.message)
    print("Errors: ", ex.errors)
except Exception as ex:
    raise ex

File: Java/marketplace/CreateTransfer.java

package examples.marketplace;

import me.pagar.api.PagarmeCoreApiClient;
import me.pagar.api.controllers.RecipientsController;
import me.pagar.api.http.client.*;
import me.pagar.api.models.*;

public class CreateTransfer {

    public static void main(String[] args) {

        String basicAuthUserName = "ak_test_4tdVXpseumRmqbo"; // The username to use with basic authentication
        String basicAuthPassword = ""; // The password to use with basic authentication

        PagarmeCoreApiClient client = new PagarmeCoreApiClient(basicAuthUserName, basicAuthPassword);

        RecipientsController recipients_controller = new RecipientsController();

        CreateTransferRequest request = new CreateTransferRequest();

        request.setAmount(100);

        String recipientId = "rp_RElaP4NMCJu08V9m";

        recipients_controller.createTransferAsync(recipientId, request, null, new APICallBack<GetTransferResponse>() {
            @Override
            public void onSuccess(HttpContext context, GetTransferResponse response) {

                System.out.println("Transfer create!");
                System.out.println("Transfer ID: " + response.getId());
                System.out.println("Transfer status: " + response.getStatus());

            }

            @Override
            public void onFailure(HttpContext context, Throwable error) {

                System.out.println("Status response: " + context.getResponse().getStatusCode());
                System.out.println(error.getMessage());
                error.printStackTrace();

            }
        });
    }
}